src/ucp/wireup/select.c

```diff
@@ -1701,6 +1731,7 @@ ucp_wireup_add_tag_lane(const ucp_wireup_select_params_t *select_params,
     criteria.title          = "tag_offload";
     criteria.local_md_flags = UCT_MD_FLAG_REG; /* needed for posting tags to HW */
     criteria.calc_score     = ucp_wireup_am_score_func;
+    criteria.lane_type          = UCP_LANE_TYPE_TAG;
```

Comment:
valign

Reply:
right, I ran clang only on latest commit each time, but it's better to run on all commits each time.